clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -96,3 +96,16 @@ if(WIN32)
 else()
     set(DNN_LIB "dnnl")
 endif()
+
+# Link MKL library to target
+macro(DPCT_HELPER_ADD_MKL_TO_TARGET target)
+  if(WIN32)
+    target_compile_options(${target} PUBLIC -fsycl /DMKL_ILP64 /Qmkl:parallel /Qtbb /MD)
```

Comment:
This line still cannot define `MKL_ILP64` in all related obj files.
Could you prepare a full project case in https://github.com/intel-sandbox/cmake_script_migration which meet below requirements:
1. this project has 3 files, a.cu, b.cu and c.cu
2. a.cu and b.cu contains cufft API, c.cu does not contain cufft API
3. a.cu b.cu c.cu are compiled to a.o b.o c.o first, then a.o b.o c.o are linked together with cufft to culib by cuda_add_cufft_to_target